modules/common/react/lib/Popper.tsx

```diff
@@ -69,9 +69,15 @@ export class Popper extends React.PureComponent<PopperProps> {
       return;
     }
 
-    this.popper = new PopperJS(this.props.anchorElement, popperNode, {
-      placement: this.props.placement,
-      ...this.props.popperOptions,
-    });
+    if (this.props.anchorElement) {
+      this.popper = new PopperJS(this.props.anchorElement, popperNode, {
+        placement: this.props.placement,
+        ...this.props.popperOptions,
+      });
+    } else {
+      console.warn(
+        `Popper: anchorElement was not defined. A valid anchorElement must be provided to render a Popper`
```

Comment:
Hmm... I don't remember why I made this change...

Reply:
But why did we make it optional in the first place?

Reply:
I'm going to try taking it out and see what happens.

Reply:
TS builds, Jest tests and Cypress tests all pass without this change. I'm reverting.

Reply:
This change is actually needed by this line: https://github.com/Workday/canvas-kit/pull/184/files#diff-a92e8ec0649e74da7d56e8238af87821R28

Changing `anchorElement` from `Element` to `Element | null` allows this component to work much better with refs without a bunch of defensive code in the render method of a consumer